task_03/script.js

```diff
@@ -0,0 +1,147 @@
+const X_CLASS = "x";
+const O_CLASS = "circle";
+const WINNING_COMBINATIONS = [
+  [0, 1, 2],
+  [3, 4, 5],
+  [6, 7, 8],
+  [0, 3, 6],
+  [1, 4, 7],
+  [2, 5, 8],
+  [0, 4, 8],
+  [2, 4, 6]
+];
+
+// Board
+const board = document.getElementById("board");
+const cellElements = document.querySelectorAll(".board__cell");
+
+const winningMessageElement = document.getElementById("winning-message");
+const winningMessageTextElement = document.getElementById("winning-message-text");
+const winnerNameInput = document.getElementById("winner-name");
+const winnerList = document.getElementById("sidebar-winner-list");
```

Comment:
Why is this constant defined here instead of in the scope of the function renderWinnerListItem()?